projectroles/app_settings.py

```diff
@@ -626,6 +626,44 @@ def delete(cls, app_name, setting_name, project=None, user=None):
             )
         )
 
+    @classmethod
+    def delete_settings_by_scope(
```

Comment:
This is a good idea. I would like the following minor changes:

- Rename method into `delete_by_scope()`
- Note in the docstring that this deletes all settings within a scope for a specific project/user combination
- Change signature so that `scope` comes before `project` and `user`
- Remove the `=None` from `scope` as it must always be set
- Add `SITE` scope support perhaps? :)